src/System.IO.Ports/tests/System.IO.Ports.Tests.csproj

```diff
@@ -5,8 +5,6 @@
     <ProjectGuid>{4259DCE9-3480-40BB-B08A-64A2D446264B}</ProjectGuid>
     <TestCategories>InnerLoop;OuterLoop</TestCategories>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Debug|AnyCPU'" />
```

Comment:
We shouldn't remove this because Configurations.props still have a netfx configuration. Plus it has nothing related to your changes.